Makefile

```diff
@@ -37,26 +37,49 @@ CLIENT_SRC_FILE:=$(CLIENT_DIR)/Containerfile.fedora
 TOOLBOX_SRC_FILE:=$(TOOLBOX_DIR)/Containerfile.fedora
 
 TAG?=latest
-SERVER_NAME:=samba-container:$(TAG)
-NIGHTLY_SERVER_NAME:=samba-container:nightly
-AD_SERVER_NAME:=samba-ad-container:$(TAG)
-NIGHTLY_AD_SERVER_NAME:=samba-ad-container:nightly
-CLIENT_NAME:=samba-client-container:$(TAG)
-TOOLBOX_NAME:=samba-toolbox-container:$(TAG)
-
-SERVER_REPO_NAME:=quay.io/samba.org/samba-server:$(TAG)
-NIGHTLY_SERVER_REPO_NAME:=quay.io/samba.org/samba-server:nightly
-AD_SERVER_REPO_NAME:=quay.io/samba.org/samba-ad-server:$(TAG)
-NIGHTLY_AD_SERVER_REPO_NAME:=quay.io/samba.org/samba-ad-server:nightly
-CLIENT_REPO_NAME:=quay.io/samba.org/samba-client:$(TAG)
-TOOLBOX_REPO_NAME:=quay.io/samba.org/samba-toolbox:$(TAG)
-
-BUILDFILE_SERVER:=.build.server
-BUILDFILE_NIGHTLY_SERVER:=.build.nightly-server
-BUILDFILE_AD_SERVER:=.build.ad-server
-BUILDFILE_NIGHTLY_AD_SERVER:=.build.nightly-ad-server
-BUILDFILE_CLIENT:=.build.client
-BUILDFILE_TOOLBOX:=.build.toolbox
+NIGHTLY_TAG?=nightly
+
+OS_NAME?= fedora
+
+
+_REPO_BASE?=quay.io//samba.org/
+SERVER_NAME:=samba-$(OS_NAME)-server-container:$(TAG)
+NIGHTLY_SERVER_NAME:=samba-$(OS_NAME)-server-container:$(NIGHTLY_TAG)
+AD_SERVER_NAME:=samba-$(OS_NAME)-ad-server-container:$(TAG)
+NIGHTLY_AD_SERVER_NAME:=samba-$(OS_NAME)-ad-server-container:$(NIGHTLY_TAG)
+CLIENT_NAME:=samba-$(OS_NAME)-client-container:$(TAG)
```

Comment:
I'm not too keen on changing the image names. I'll elaborate more in the summary.